assignment-v2/test-case-7.json

```diff
@@ -0,0 +1,37 @@
+{
+  "experiment": "empty_string_variations_test",
```

Comment:
Tests in this folder are for the old RAC SDKs which are no longer supported so I don't think we need to bother with this. (and those use rac-experiments-v3.json)